tox.ini

```diff
@@ -4,13 +4,12 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py{27,34,35,36,py,py3}-pytest{29,30}, flake8
+envlist = py{27,34,35,36,py,py3}, flake8
 
 [testenv]
-commands = py.test {posargs}
+commands = pytest {posargs}
 deps =
-    pytest29: pytest==2.9.2
-    pytest30: pytest==3.0.1
+    pytest==3.7.4
```

Comment:
If we are running only against one pytest version, I believe it should be the last one.

Reply:
See the PR comment: it appears to fail with 3.8

Reply:
Oops my bad, sorry! 😅